lib/spack/env/cc

```diff
@@ -173,20 +213,37 @@ preextend() {
     unset IFS
 }
 
-# system_dir PATH
+# contains str listval [sep=:] [strip=/]
```

Comment:
```suggestion
# contains ELEMENT LISTVAL [SEP=:] [STRIP=/]
```
I spent a while misunderstanding this method because I got the order of arguments confused, I think this change to the comment would help, and changed to all-caps to match other docstring comments.